QuasistaticBrownianThermalNoise/QuasistaticBrownianThermalNoise.cpp

```diff
@@ -605,6 +683,32 @@ ElasticProblem<dim>::ElasticProblem()
     }
   }
 
+  // Set Y_Ta2O5 in terms of the bulk and shear moduli
```

Comment:
Repeating code. Please make a function that takes the name of an iso material and returns the scaled yijkl, then use it for each case. Make another function that does the same for cubic crystal materials.

Reply:
Please file an issue to implement this change. I want to get going with science results, so I'm ok with the repeated code for now.